cromshell

```diff
@@ -664,6 +664,25 @@ function submit()
 
   assertRequiredFileIsNonEmpty "${wdl}" "WDL"
   assertRequiredFileIsNonEmpty "${json}" "Input JSON"
+
+  # At this point, we should validate our inputs if we can:
+  which womtool &> /dev/null
+  r=$?
+  if [[ ${r} -eq 0 ]] ; then
+    echo -ne "Validating WDL and JSON before submission..."
+		local tmpValidationOutput=$( makeTemp )
+    womtool validate ${wdl} -i ${json} &> ${tmpValidationOutput}
+    if [[ $? -eq 0 ]] ; then
+      echo -e '\tSuccess!'
+    else
+      echo -e '\tFailure!'
```

Comment:
Probably don't need this failure AND the subsequent error message.

Reply:
Fixed!